p2p_wallet/Scenes/Main/Home/Subviews/Banner/Home.BannerSection.swift

```diff
@@ -29,6 +31,11 @@ extension Home {
                 ),
                 viewModel: viewModel
             )
+            
+            viewModel
```

Comment:
Should we move the data change listener to BEStaticSectionsCollectionView.Section? The library should handle automatically the changes from view model.